Tests/swiftlintTests/Helpers/SwiftLintExecutable.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SwiftLintExecutable.swift
+//  SwiftLint
+//
+//  Created by Sash Zats on 6/22/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import Result
+
+
+enum SwiftLintExecutionError: ErrorType {
+    case ExecutionFailed
+    case LintError(String)
+}
+
+struct SwiftLintExecutable {
+
+    private let path: String
+
+    init() {
+        let products = testBundle.bundleURL.URLByDeletingLastPathComponent!
+        let executable = products.URLByAppendingPathComponent("swiftlint")
```

Comment:
Thanks a lot! That fixed the Xcode part of the tests. 👍 
I'll take a look at SPM failure